tree_entry.go

```diff
@@ -109,9 +109,10 @@ func (tes Entries) Sort() {
 }
 
 type commitInfo struct {
-	id    string
-	infos []interface{}
-	err   error
+	id        string
```

Comment:
Since `id` was only used for identify the object, and now we use `entryName`, I think it can be removed.